config/setupTests.js

```diff
@@ -83,4 +88,30 @@ global.window.__scalprum__ = {
 };
 
 jest.useFakeTimers('modern').setSystemTime(new Date('2020-01-01').getTime());
+  
+jest.mock('@redhat-cloud-services/frontend-components/Inventory', () => ({
+    InventoryTable: (props) => <div {...props} />,
+}));
 
+jest.mock('@redhat-cloud-services/frontend-components/AsyncComponent', () => {
+    return {
+        __esModule: true,
+        default: () => (<div>AsyncComponent</div>)
+    };
+});
+
+jest.mock('@redhat-cloud-services/frontend-components-pdf-generator', () => {
```

Comment:
I'm still wondering why we need to mock the pdf-generator now.

Reply:
Vuln is still using the old PDF generation, we had the new one implemented, but we rolled back due to some issues

Reply:
@leSamo But why do we have to start mocking it now in this PR here and master passes tests without it. That is what I am wondering.

Reply:
No idea, we had to mock the InventoryTable as well :man_shrugging:

Reply:
@AsToNlele Mind just giving a quick check and run the tests without it and see what and why it fails?

Reply:
![image](https://github.com/RedHatInsights/vulnerability-ui/assets/20592948/ba62910e-f43e-4982-ae71-74d0ef631a26)
It's the same error we faced with `InventoryTable`